dev/ldap/README.md

```diff
@@ -0,0 +1,72 @@
+# Local Lightweight Directory Access Protocol (LDAP) Setup
```

Comment:
This is really cool stuff - can we put it under a `docker/` top-level folder instead of `dev/`? I think that makes the purpose clearer.